src/models/workspaces.ts

```diff
@@ -7,6 +7,10 @@ export type WorkspaceMode = typeof WorkspaceModeEnum.Type;
 export const WorkspaceToolEnum = types.enumeration("tool", ["delete", "geometry", "select", "text"]);
 export type WorkspaceTool = typeof WorkspaceToolEnum.Type;
 
+export const kSectionID = "section";
```

Comment:
We're not consistent on our constant styling, or our capitalization of ID - I copied conventions from other similar variables in this case.